qemu/tests/verify_panic_status_with_pvpanic.py

```diff
@@ -0,0 +1,41 @@
+import logging
+
+
+def run(test, params, env):
+    """
+    Verify the QMP even with -device pvpanic when trigger crash,this case will:
+
+    1) Start VM with pvpanic device.
+    2) Check if pvpanic device is exist in guest.
+    3) Trigger crash in guest.
+    4) Check vm status with QMP.
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environmen.
+    """
+
+    qmp_command = params["qmp_command"]
+    stop_kdump_command = params["stop_kdump_command"]
+    trigger_crash = params["trigger_crash"]
+    qmp_check_info = params["qmp_check_info"]
+
+    vm = env.get_vm(params["main_vm"])
+    session = vm.wait_for_login()
+    qmp_port = vm.monitor
+
+    if params.get("check_info"):
+        qtree_info = vm.monitor.info("qtree")
+        if params.get("check_info") not in qtree_info:
+            test.fail("Not find pvpanic device in guest")
+
+    try:
+        session.cmd(stop_kdump_command)
+        session.cmd(trigger_crash, timeout=10)
+    except Exception as detail:
+        logging.info(detail)
```

Comment:
Here should tolerant only certain type exception not all, you might miss some bz this way.